tooling/FST-1029-langversion and preview library functions.md

```diff
@@ -0,0 +1,53 @@
+
+The design suggestion [](https://github.com/dotnet/fsharp/issues/5496) has been marked "approved in principle".
+This RFC covers the detailed proposal for this suggestion.
+
+* [ ] Approved in principle
+* [ ] [Suggestion](https://github.com/fsharp/fslang-suggestions/issues/825)
+* [ ] Implementation: [In progress]https://github.com/dotnet/fsharp/pull/8042)
+
+
+# Summary
+[summary]: Allow --langversion:preview to disable compiler warnings from APIs using the ExperimentalAttribute().
+
+# Motivation
+[motivation]: Shipping preview versions of the F# language early is very difficult, because there isn't a delivery mechanism that is seperate from Visual Studio or the DotNet Cli.
```

Comment:
```suggestion
Shipping preview versions of the F# language early is very difficult, because there isn't a delivery mechanism that is separate from Visual Studio or the .NET CLI.
```